### ISSUE TITLE
Raise an error when adding an aggregate tx as an inner transaction of another aggregate tx

### ISSUE THREAD

**Issue by dgarcia360** [nemtech/nem2-sdk-typescript-javascript#38]
**Expected behavior**

```
Scenario: An account tries to create an escrow contract inside an escrow contract
   Given Alice defined an escrow contract inside an escrow contract
    And "Alice" locked 10 "xem" to guarantee that the contract will conclude in less than 2 days
    When she publishes the contract
    Then she should receive the error "Not possible to nest aggregate transactions"
```
**Actual behavior**

After the transaction is announced, it never gets confirmed. However, the sender account is not notified about the error through the status channel.

To discuss if it's better to raise this error in the API/ P2P or SDK layer.

**Comment by evias** [comment]
I believe the SDK should definitely raise this kind of error.

Regarding the API/P2P layers raising the error, has there been progress in the discussion?

**Comment by evias** [comment]
Fixed and merged in v0.11
